plugins/modules/zypper_repository.py

```diff
@@ -32,13 +33,24 @@
         type: str
     repo:
         description:
-            - URI of the repository or .repo file. Required when state=present.
+            - URI of the repository or .repo file. Required when I(state=present) and I(action=add).
```

Comment:
```suggestion
            - URI of the repository or .repo file. Required when O(state=present) and O(action=add).
```
(See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation)

Reply:
updated